tests/integration/test_k8s_with_cluster.py

```diff
@@ -522,42 +497,13 @@ def test_cli_secret_handler_crud(test_namespace: str):
     assert process_four.returncode == 0
 
 
-def test_deployment_will_not_be_created_if_namespace_is_not_setup_for_bodywork(
-    random_test_namespace: str,
-):
-    process_one = run(
-        [
-            "bodywork",
-            "deployment",
-            "create",
-            f"--namespace={random_test_namespace}",
-            "--name=bodywork-test-project",
-            "--git-repo-url=https://github.com/bodywork-ml/bodywork-test-project",
-        ],
-        encoding="utf-8",
-        capture_output=True,
-    )
-    assert f"{random_test_namespace} is not setup for use" in process_one.stdout
-    assert process_one.returncode == 1
-
-
 def test_deployment_of_remote_workflows(random_test_namespace: str):
```

Comment:
Doesn't need `random_test_namespace` now?

Reply:
Nope because the namespace is static i.e. it is obtained from the config.YAML